src/robusta/core/sinks/sink_base_params.py

```diff
@@ -91,12 +91,21 @@ def validate_exactly_one_defined(cls, values: Dict):
 class GroupingParams(BaseModel):
     group_by: GroupingAttributeSelectorListT = ["cluster"]
     interval: int = 15*60  # in seconds
+    aligned: bool = False
     notification_mode: Optional[NotificationModeParams]
 
     @root_validator
-    def validate_notification_mode(cls, values: Dict):
-        if values is None:
-            return {"summary": SummaryNotificationModeParams()}
+    def validate_interval_alignment(cls, values: Dict):
+        if values["aligned"]:
+            if values["interval"] < 24 * 3600:
+                if (24 * 3600) % values["interval"]:
```

Comment:
not sure i understand, we allow the interval to be seconds but we dont support seconds, shouln't we change it to minutes or have it round up to the nearest minute instead of failing ?

Reply:
There will be no backwards compatibility problems, as prior users don't use the interval aligning mechanism (it's not been released yet).

Also, we support aligning on thresholds shorter than a minutes here without problem. For example, if the user specs `interval` as 15 seconds, the intervals will be automatically aligned at `x:00`, `x:15`, `x:30`, `x:45` (for consecutive minutes `x`).

Reply:
got it